dumbot.py

```diff
@@ -291,6 +291,9 @@ class Bot:
 
         timeout (`int`):
             The timeout, in seconds, to use when fetching updates.
+        
+        test_mode (`bool`):
+            Use bot in test DC
```

Comment:
This could use a longer explanation. What is the test DC? How do I create the bot there? How do I interact with it? Also please add a period at the end of the sentence.

Reply:
oh thanks for the reply lomi, ill add a longer detailed explanation today